apps/wallet-mobile/src/features/Receive/common/ShareQRCodeCard/ShareQRCodeCard.tsx

```diff
@@ -98,10 +98,12 @@ export const ShareQRCodeCard = ({content, title, isCopying, onLongPress}: ShareQ
 }
 
 const useStyles = () => {
-  const screenWidth = useWindowDimensions().width
+  const {width: screenWidth, height: screenHeight} = useWindowDimensions()
   const {theme} = useTheme()
   const {color, typography} = theme
 
+  const cardSpacing = screenHeight > 800 ? 32 : 16
```

Comment:
let's add later to the theming these breakpoints, for now please add as constant the 800 -> breakpointLarge

Reply:
to avoid magic numbers